nupic/algorithms/anomaly_likelihood.py

```diff
@@ -103,6 +103,19 @@ def __init__(self, claLearningPeriod=300, estimationSamples=300):
     self._reestimationPeriod = 100 
 
 
+  def __eq__(self, o):
+    if not isinstance(o, AnomalyLikelihood):
+      return False
+    elif (self._iteration == o._iteration and
+      self._historicalScores == o._historicalScores and
+      self._distribution == o._distribution and
+      self._probationaryPeriod == o._probationaryPeriod and
+      self._claLearningPeriod == o._claLearningPeriod):
+      return True
+    else:
+      return False
+
```

Comment:
applied, thanks! 
Btw, I recall we had some code for "ObjectEquals" that did just this, compared objects as equality of all their inner members. I think the code went away during some cleanups, do you  think it would be usable? (aspirants that come on my mind: MovingAverage, Anomaly, ...)